de.philippun1.turtlegit.json

```diff
@@ -0,0 +1,82 @@
+{
+  "app-id": "de.philippun1.turtlegit",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "turtlegit_cli",
+  "finish-args" : [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=home"
+  ],
+  "cleanup" : [
+    "/bin/git2",
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "app-install",
```

Comment:
the main module, that usually has the name of the app (`turtlegit` for example) is also usually last since the order is the dependencies order.